ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/methods/privacy/EeaDeletePrivacyGroup.java

```diff
@@ -46,8 +46,8 @@ public String getName() {
   public JsonRpcResponse response(final JsonRpcRequest request) {
     LOG.trace("Executing {}", RpcMethod.EEA_DELETE_PRIVACY_GROUP.getMethodName());
 
-    final String privacyGroupId = parameters.required(request.getParams(), 1, String.class);
```

Comment:
This seems like a non-change.  If there are any other alterations can this be reverted?